covalent_dispatcher/_cli/service.py

```diff
@@ -386,20 +388,55 @@ def status() -> None:
 
 
 @click.command()
-def purge() -> None:
+@click.option("-H", "--hard", is_flag=True, help="Perform a hard purge, deleting the DB as well")
```

Comment:
Help text: "Delete the dispatch database upon purge"

Also set `show_default=True` for these options.

Reply:
Here they have mentioned that if the default is False then even `show_default` will not show the default value: https://click.palletsprojects.com/en/8.1.x/options/#basic-value-options